bazel/closure_grpc_web_library.bzl

```diff
@@ -55,12 +55,12 @@ def _generate_closure_grpc_web_srcs(
         mode,
         sources,
         transitive_sources):
-    all_sources = [src for src in sources] + [src for src in transitive_sources]
+    all_sources = [src for src in sources] + [src for src in transitive_sources.to_list()]
     proto_include_paths = [
         "-I%s" % p
         for p in _proto_include_paths(
             [f for f in all_sources],
-        )
+        ).to_list()
```

Comment:
I think you could change `_proto_include_paths` to return a list instead. Not sure why it returns a depset...

Reply:
Done!